src/components/icon/icon.tsx

```diff
@@ -29,12 +52,23 @@ interface IconBaseState {
 }
 
 const getSpecificIcon = (props: any) => ({
+    access_time: <AccessTime className={props.className} />,
```

Comment:
If we use enum to get specific icon, then I think that we should use it as keys too. For example:
```
[IconTypes.ACCESS_TIME]: <CloseAnnouncement className={props.className} />,
```